### DISCUSSION TITLE
Get the Mica transparency level from WinUI 3

### DISCUSSION THREAD

**Discussion by onionware-github** [amwx/FluentAvalonia#320] [Q&A]
Hi, I want to add Mica to my FluentAvalonia app, but I dont get the right transparency for my page background to achieve the same colors that my WinUI 3 version of my app has:
I use the WindowsUIComposition and Background="Transparent" in my MainWindow.
Here are screenshots from the Avalonia version:
![image](https://user-images.githubusercontent.com/91136545/226181006-5f427abe-7cde-4e0f-8445-bfb84557d452.png)
![image](https://user-images.githubusercontent.com/91136545/226181007-f43a5bdf-1d6e-4db1-bc11-a02785ed6036.png)

Here are screenshots of the WinUI 3 version of my app:
![image](https://user-images.githubusercontent.com/91136545/226181042-1b58f83c-e696-4122-9b28-bd575adfe745.png)
![image](https://user-images.githubusercontent.com/91136545/226181043-93fa4d0d-cc5c-4e93-b150-bd7455e0a513.png)

You can see that the WInUI version only uses a little amount of Mica transparency, but the Avalonia version has too much Mica.
The Avalonia version should have the same style as the WinUI 3 version, does anybody know the correct values, that I have to use as my page background to achieve the same look?

**Comment by amwx** [comment]
FWIW I've opened an issue upstream about this: https://github.com/AvaloniaUI/Avalonia/issues/10719

The Mica brush isn't properly created upstream, it just pulls the blurred wallpaper from the WinUI compositor and returns. Unfortunately, this likely won't be fixed anytime soon given restrictions upstream until 11.0 is released. In the meantime you can try to do what I do in the sample app:
https://github.com/amwx/FluentAvalonia/blob/8636ee6ed41420ea93b93a1b6011999768330f83/samples/FluentAvaloniaSamples/Views/MainWindow.axaml.cs#L143-L168

It doesn't look very good, but its something.
